stixpy/vis/aia_reprojection.py

```diff
@@ -0,0 +1,235 @@
+from datetime import datetime
+from stixcore.ephemeris.manager import Position
+import stixcore.data.test
+from pathlib import Path
+import sunpy
+import sys
```

Comment:
The should be separated into 3 groups python standard library, 3rd part e.g numpy, sunpy etc and then the last section is import from stixcore it's self. 

There is a plugin which should do this for you automatically isort I need to check whey it not working.